src/Extension/FluentExtension.php

```diff
@@ -679,6 +685,86 @@ public function onAfterDuplicate($original, $doWrite, $relations): void
                 SQLUpdate::create($localisedTable, $fieldsToUpdate, ['RecordID' => $toID])->execute();
             }
         }
+
+        // We don't have any localised relations to cover
+        if (count($copyHasOneRelations) === 0) {
```

Comment:
```suggestion
        if (empty($copyHasOneRelations)) {
```
This is just a personal preference so feel free to leave as is if you want.

Reply:
Will leave this as it as it's just a preference, personally I don't like using empty() as it's too greedy (will work on many variable types).